rust/core/src/address.rs

```diff
@@ -1,31 +1,1041 @@
-#[derive(Clone, Debug, serde::Deserialize, serde::Serialize, schemars::JsonSchema)]
-pub struct Address {
-    #[serde(skip)]
-    pub encodings: Option<AddressEncoding>,
+use super::*;
+use schemars::JsonSchema;
+use cbor_event::{de::Deserializer, se::Serializer};
+use bech32::ToBase32;
+//use crate::byron::{ProtocolMagic, ByronAddress, ByronAddressError};
```

Comment:
do we need that here?

Reply:
We will when we bring over the byron module.